addons/repair/models/stock_move.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class StockMove(models.Model):
```

Comment:
Look in the `repair.py` file, there is already an inherit for `stock.move` model and the `repair_id` field already exist too.
I think it's good to create its own file by model, but so you should clean the existing one.

Reply:
@svs-odoo Ok, I let it the way it was.